### ISSUE TITLE
kkolur3

### ISSUE THREAD

**Issue by kkolur3** [RoboJackets/jr#28]
# Week 7: Git Challenge

Welcome to week 7 of software training! This week, you'll be completing a series of `git` and GitHub challenges. **To get started, type your GT username into the Title box above, then click the "Submit new issue" at the bottom of this page!** From there, work through the rest of this checklist.

If you need any help, we recommend these online resources:
- https://help.github.com/articles/github-glossary/
- https://try.github.io/levels/1/challenges/1
- https://learngitbranching.js.org/
- https://chris.beams.io/posts/git-commit/

You can, of course, also flag down an instructor if you run into anything.

## Your objectives

Replace `gburdell3` with your GT username anywhere it appears in this list.

- [x] 1. Star this repository.
- [x] 2. Create a fork of this repository.
- [x] 3. Clone your fork to your computer.
- [ ] 4. Set your forked repository as a remote named `origin`.
- [ ] 5. Set our repository as a remote named `rj`.
- [ ] 6. Create a branch named your GT username (ex. `gburdell3`).
- [ ] 7. Use the terminal to create a folder named `2017` in the root of the repo.
- [ ] 8. Use the terminal to create an empty text file named `gburdell3.txt` in the `2017` folder.
- [ ] 9. Stage and commit the file (make sure you leave good commit messages!)
- [ ] 10. Create a new branch from `gburdell3` named `gburdell3-add-text`.
- [ ] 11. Add some ASCII art to your file (no longer than 30 lines, no more than 80 characters per line) on that branch and merge it into the `gburdell3` branch. Use spaces, not tabs, for indentation in your art, as tabs may distort your image in jr.
- [ ] 12. Make a merge conflict happen between `gburdell3` and a branch named `gburdell3-merge-conflict`.
- [ ] 13. Resolve the merge conflict.
- [ ] 14. Create two branches named `b1` and `b2`.
- [ ] 15. Commit a file called `stuff1.txt` in `b1`.
- [ ] 16. Commit a file called `stuff2.txt` in `b2`.
- [ ] 17. Take the changes from `b1` and apply them to `b2`.
- [ ] 18. In `b2`, delete both files and commit the changes.
- [ ] 19. Merge `b2` into `b1`.
- [ ] 20. Merge `b1` into `gburdell3` (this should not cause a merge conflict).
- [ ] 21. Send your committed changes to the `gburdell3` branch on your fork
- [ ] 22. Create a pull request from your fork’s `gburdell3` branch to our master branch and ask one of the instructors in your session to approve it (if you try to merge it you won’t be able to because permissions). **In your pull request description, add a line that says "Fixes #xx" where #xx is the number of the issue you created.** This will auto-close your issue once your pull request is merged.
- [ ] 23. After your PR has been approved, delete all local branches except for `master` and `gburdell3`.
